ros2param/ros2param/verb/set.py

```diff
@@ -57,7 +57,7 @@ def main(self, *, args):  # noqa: D102
 
         with DirectNode(args) as node:
             parameter = Parameter()
-            Parameter.name = args.parameter_name
+            parameter.name = args.parameter_name
```

Comment:
just curious  that this is bug? did it work okay?

Reply:
That's a good question.  I just noticed it while looking at the code.

It seemed to have worked fine before, but I think that was because we were setting the `name` field on the `Parameter` *class*, not the object.  The proper thing to do here is to set it on the object itself, which is what this change does.  Does that make sense?